### DISCUSSION TITLE
Possibility to insert the last changes date that occures on the page

### DISCUSSION THREAD

**Discussion by didntchooseaname** [retypeapp/retype#596] [Ideas]
Good morning, It might be interesting to add an automatic date feature at the bottom of the page to show the recency of the page in question (date modified), based on the date of the repo commit to that page and a link to it.

**Comment by geoffreymcgill** [comment]
We would love to have the _last updated_ data included on the page, but we have never been able to figure out a way to make it work without making remote calls and over complicating the process.

See the following issues:

https://github.com/retypeapp/retype/issues/383#issuecomment-1211561339

https://github.com/retypeapp/retype/issues/503
